terraform/resources/aca/locals.tf

```diff
@@ -135,41 +119,69 @@ locals {
           value = var.azure_container_name
         },
         {
-          name  = "NEXT_PUBLIC_NON_INTEGRATED_VIEWER",
-          value = false
+          name  = "CONFIG_NAME",
+          value = var.ecr_viewer_mode
+        },
+        {
+          name  = "NBS_API_PUB_KEY"
+          value = var.nbs_api_public_key
+        },
+        {
+          name  = "NBS_PUB_KEY"
```

Comment:
Does this need a comma?

Reply:
Strangely, no! This is the identical code from `dibbs-tf-envs`, and it works just fine in all of our environments. Looks like commas are optional for this data structure. That said, I will gladly add a couple in for consistency. Great catch!